chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -272,7 +272,9 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
     useEffect(() => {
         if (state?.appStates?.hideStartChatButton === true) {
             //handle OOH pane
-            if (state.appStates.outsideOperatingHours === true) {
+            // Only set OutOfOffice state for new conversations, allow existing Active/InActive conversations to continue
+            if (state.appStates.outsideOperatingHours === true && 
+                state.appStates.conversationState !== ConversationState.Active) {
```

Comment:
This will set OOH when prechat is present , Loading pane is present , proactive chat and also during post survey load 

do we need to do it only when conversation is Closed?

export enum ConversationState {
    Prechat = 0,
    Loading = 1,
    ReconnectChat = 2,
    OutOfOffice = 3,
    ProactiveChat = 4,
    Active = 5,
    InActive = 6,
    PostchatLoading = 7,
    Postchat = 8,
    Closed = 9,
    Error = 10
}